.github/workflows/demo_apps_build.yaml

```diff
@@ -0,0 +1,154 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Telink
+
+on:
+    push:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+env:
+    CHIP_NO_LOG_TIMESTAMPS: true
+
+jobs:
+    telink:
+        name: Telink
+        env:
+            BUILD_TYPE: telink
+            ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+            ZEPHYR_SDK_VERSION: 0.15.2
+            ZEPHYR_SDK_INSTALL_DIR: zephyr_sdk/zephyr-sdk-"${ZEPHYR_SDK_VERSION}"
+
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: .
+        if: github.actor != 'restyled-io[bot]'
+
+        steps:
+            - uses: Wandalen/wretry.action@v1.0.36
+              name: Checkout
+              with:
+                  action: actions/checkout@v3
+                  with: |
+                      token: ${{ github.token }}
+                  attempt_limit: 3
+                  attempt_delay: 2000
+
+            - name: Install dependencies
+              run: |
+                    sudo apt-get install -fy --no-install-recommends ninja-build cmake make gcc gcc-multilib g++-multilib libsdl2-dev
+
+            - name: Setup toolchain
+              run: |
+                    mkdir zephyr_sdk \
+                    && cd zephyr_sdk \
+                    && wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v"${ZEPHYR_SDK_VERSION}"/zephyr-sdk-"${ZEPHYR_SDK_VERSION}"_linux-x86_64.tar.gz \
+                    && wget -O - https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v"${ZEPHYR_SDK_VERSION}"/sha256.sum | shasum --check --ignore-missing \
+                    && tar xvf zephyr-sdk-"${ZEPHYR_SDK_VERSION}"_linux-x86_64.tar.gz \
+                    && rm -rf zephyr-sdk-"${ZEPHYR_SDK_VERSION}"_linux-x86_64.tar.gz \
+                    && cd zephyr-sdk-"${ZEPHYR_SDK_VERSION}" \
+                    && ./setup.sh -t riscv64-zephyr-elf -h -c
+
+            - name: Setup Zephyr sources
+              run: |
+                    echo "West setup" \
+                    && python -m venv .venv \
+                    && source .venv/bin/activate \
+                    && python -m pip install -U pip \
+                    && python -m pip install -U --no-cache-dir west \
+                    && west init -l \
+                    && west update \
+                    && west zephyr-export \
+                    && pip install -r scripts/requirements.txt \
+                    && source zephyr-env.sh \
+                    && west blobs fetch hal_telink \
+                    && ls -la \
```

Comment:
Remove "ls -la" and "pwd" in all places

Reply:
Ah, yes, these were for some checks. Done